<launch>
  <!-- Arguments -->
  <arg name="model" default="burger" doc="model type [burger, waffle, waffle_pi]"/>
  <arg name="cmd_vel_topic" default="/cmd_vel" />
  <arg name="odom_topic" default="odom" />
  <arg name="move_forward_only" default="false"/>

  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find warehouse_robot)/config/costmap_common_params_$(arg model).yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find warehouse_robot)/config/costmap_common_params_$(arg model).yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find warehouse_robot)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find warehouse_robot)/config/global_costmap_params.yaml" command="load" />
    
    <!-- Costmap parameters -->
    <param name="local_costmap/inflation_radius" value="0.1" />
    <param name="global_costmap/inflation_radius" value="0.2" />
  </node>
</launch>
